tests/test-data-2.csv

```diff
@@ -0,0 +1,27 @@
+CAT_ID,f480w_corr_1,f390w_corr_1
```

Comment:
Let's separate out data files in their own folders

Reply:
Done!